app/views/notification_mailer/ticket_assigned.text.erb

```diff
@@ -5,7 +5,7 @@
 <%= t('notify_ticket_assigned', default: 'You have recently been assigned to the following topic(s):') %>
 
 <% @bulk_topics.each do |topic| %>
-  <%= topic.name %>
+  <%= topic.name + '  '%><%= link_to t('view_on_site', default: "View on Site"), topic %>
```

Comment:
Oh, dang! 
Yeah, thanks.

Also, I think completely removing the `Reply directly to this email or view this online` may be better since each of the assigned topics already have a `View on Site` link, and this will be the save as the link for the reply even when the assignment is for 1 topic.
Thoughts?